<!--#######################################-->
<!-- Copyright (c) 2024  Dong Zhaorui.     -->
<!-- All rights reserved.                  -->
<!-- Author: Dong Zhaorui 847235539@qq.com -->
<!-- Date  : 2024-01-17                    -->
<!--#######################################-->

<launch>

    <!-- map broadcaster -->
    <node pkg="hex_map_3d" type="map2d_broadcaster.py" name="map2d_broadcaster" output="screen" >
        <param name="yaml_path" value="$(find hex_map_3d)/output/default/map2d/map.yaml" />
        <param name="trinary_flag" value="false" />
    </node>

</launch>